apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V92__add_clm_grp_index_correction.sql

```diff
@@ -0,0 +1,24 @@
+-- ADD CLM_GRP_ID INDEX
+--
+-- This flyway script is intended to add an index on clm_grp_id for a handful
+-- of tables that need their clm_grp_ids cleaned up for certain synthetic data.
+-- This index is temporary and will be removed in a subsequent script.
+
+-- This is done in psql only since this index is only required for a one-time
+-- cleanup script against the actual database. This script corrects the previous
+-- script which did not have unique names for each index.
+
+-- =====================
+-- inpatient_claims
+-- =====================
+${logic.psql-only} create index concurrently if not exists clm_grp_id_index_2 on public.inpatient_claims(clm_grp_id);
```

Comment:
These are fine since they are transient and will be deleted; for more permanent indexes, and to avoid namespace issues we've sort of adopted the following convention:

<table_name>_<column_name>_idx

For example, adding an index to the carrier_claims table, on the bene_id column, we'd end up with:
_carrier_claims_bene_id_idx_

Reply:
Noted!